SConstruct

```diff
@@ -157,6 +164,12 @@ if unknown:
 
 print("Building for architecture " + env["arch"] + " on platform " + env["platform"])
 
+env.Append(CPPDEFINES=env.get("cppdefines", ""))
```

Comment:
We're going to have to be explicit about this being an array by calling `split()`, I'm running into issues where a quoted list of defines is being interpreted as one single define in some environments (mostly in GitHub actions, see https://github.com/W4Shared/godot-nir-builds/actions/runs/7474876144/job/20341918395#step:6:34)

I'm going to push an update for this in a moment.